config/Run3_2022/samples.yaml

```diff
@@ -207,269 +207,365 @@ DYto2Tau_MLL_6000_powheg:
   miniAOD: /DYto2L-2Jets_MLL-10to50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
   isReference: True
   crossSection: DY_NNLO_QCD+NLO_EW
-DYto2L_M_10to50_amcatnloFXFX_nanoMuMu:
+  dirName: DYto2L_M_10to50_amcatnloFXFX
+  fileNamePattern: ^nanoEE_\d+\.root$
+DYto2Mu_M_10to50_amcatnloFXFX:
   sampleType: DY
   crossSectionStitch: DYto2L_M_10to50_amcatnloFXFX
   generator: amcatnloFXFX
   miniAOD: /DYto2L-2Jets_MLL-10to50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
   isReference: True
   crossSection: DY_NNLO_QCD+NLO_EW
-DYto2L_M_10to50_amcatnloFXFX_nanoTauTau:
+  dirName: DYto2L_M_10to50_amcatnloFXFX
+  fileNamePattern: ^nanoMuMu_\d+\.root$
+DYto2Tau_M_10to50_amcatnloFXFX:
   sampleType: DY
   crossSectionStitch: DYto2L_M_10to50_amcatnloFXFX
   generator: amcatnloFXFX
   miniAOD: /DYto2L-2Jets_MLL-10to50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
   isReference: True
   crossSection: DY_NNLO_QCD+NLO_EW
+  dirName: DYto2L_M_10to50_amcatnloFXFX
+  fileNamePattern: ^nanoTauTau_\d+\.root$
 
-DYto2L_M_50_amcatnloFXFX_nanoEE:
+DYto2E_M_50_amcatnloFXFX:
   sampleType: DY
   crossSectionStitch: DYto2L_M_50_amcatnloFXFX
   generator: amcatnloFXFX
   miniAOD: /DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
   isReference: True
   crossSection: DY_NNLO_QCD+NLO_EW
-DYto2L_M_50_amcatnloFXFX_ext1_nanoEE:
+  fileNamePattern: ^nanoEE_\d+\.root$
+  dirName: DYto2L_M_50_amcatnloFXFX
+DYto2E_M_50_amcatnloFXFX_ext1:
   sampleType: DY
   crossSectionStitch: DYto2L_M_50_amcatnloFXFX
   generator: amcatnloFXFX
   miniAOD: /DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5_ext1-v1/MINIAODSIM
   isReference: True
+  fileNamePattern: ^nanoEE_\d+\.root$
   crossSection: DY_NNLO_QCD+NLO_EW
-DYto2L_M_50_amcatnloFXFX_nanoMuMu:
+  dirName: DYto2L_M_50_amcatnloFXFX
+DYto2Mu_M_50_amcatnloFXFX:
   sampleType: DY
   crossSectionStitch: DYto2L_M_50_amcatnloFXFX
   generator: amcatnloFXFX
   miniAOD: /DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
   isReference: True
   crossSection: DY_NNLO_QCD+NLO_EW
-DYto2L_M_50_amcatnloFXFX_ext1_nanoMuMu:
+  fileNamePattern: ^nanoMuMu_\d+\.root$
+  dirName: DYto2L_M_50_amcatnloFXFX
+DYto2Mu_M_50_amcatnloFXFX_ext1:
   sampleType: DY
   crossSectionStitch: DYto2L_M_50_amcatnloFXFX
   generator: amcatnloFXFX
   miniAOD: /DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5_ext1-v1/MINIAODSIM
   isReference: True
+  fileNamePattern: ^nanoMuMu_\d+\.root$
   crossSection: DY_NNLO_QCD+NLO_EW
-DYto2L_M_50_amcatnloFXFX_nanoTauTau:
+  dirName: DYto2L_M_50_amcatnloFXFX_ext1
+DYto2Tau_M_50_amcatnloFXFX:
```

Comment:
DY->tautau from the old datasets shouldn't be used -> use the newly produced ones

Reply:
No, it is the old DYto2L. Tau decays are bugged there and must not be used.